add_student.html

```diff
@@ -1,13 +1,62 @@
-<!--
-This file will provide the backbone for adding info for a single student.
--->
-<html>
-  <head>
-  <title>Add Student</title>
-  <link rel="stylesheet" type="text/css" href="css/upls_styles.css">
-  <script src="js/main_menu.js"></script>
-  </head>
-  <body onload="mainMenu()">
-    <div class="main_menu" id="main_menu"></div>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Add Student</title>
+    <script type="text/javascript"></script>
+    <script src="https://apis.google.com/js/api.js"></script>
+    <script src="Api.js"></script>

```

Comment:
See previous comment regarding file names.

Reply:
Also, don't forget to include Connor's css file and the main menu stuff. I provided the basic skeleton for almost all the pages to be consistent and avoid these errors.